src/actions.py

```diff
@@ -36,8 +36,8 @@ def on_get_admin_password(self, event: ops.ActionEvent) -> None:
             event: The event fired from get-admin-password action.
         """
         container = self.charm.unit.get_container(self.state.jenkins_service_name)
-        if not container.can_connect():
-            event.fail("Container not yet ready.")
+        if not container.can_connect() or not self.model.storages.get(self.state.storage_name):
```

Comment:
Let's check whether this guarantees access, let's use the wordpress approach for now

Reply:
Added /proc/mount check, thank you!